### ISSUE TITLE
Performance concern and RequestAnimationFrame

### ISSUE THREAD

**Issue by tleperou** [3DJakob/react-tinder-card#113]
First, thanks to the contributors of this package 

:pray: 

Let me know if I'm wrong, but could using `requestAnimationFrame` be relevant when mutating the position of the card ?

```js
const dragableTouchmove = (coordinates, element, offset, lastLocation) => {
  // ...
  window.requestAnimationFrame(() => {
    element.style.transform = translation + rotation
  })
  return newLocation
}
```

No sure how to observe the impact on the performance from the perspective of the devTools. 
From my perspective however, the improvement is quiet visible -- specially when
* interacting back after an idle time
* getting the card in & out of the viewport

For full support and backward compatibility, we shall add [a polyfill](https://gist.github.com/mrdoob/838785) or just disable it

```js
window.requestAnimationFrame 
  ? window.requestAnimationFrame(() => element.style.transform = translation + rotation)
  : element.style.transform = translation + rotation
```

**Comment by 3DJakob** [comment]
Interesting, this sounds pretty good! So you have tried running the code with that and the performance is improved? 

How exactly would this break backward compatibility?

If possible it would be great if you could make a demo just by cloning https://github.com/3DJakob/react-tinder-card-demo and adding your modified TinderCard component. If it works and don't fundamentally change anything I'll be happy to merge it.

Another sidenote:
I have planned since long ago to update the web version to use react-spring just as the native version does. I think it would vastly improve performance, and handling and make the code bases more similar. I will do this as soon as I get time for it.

**Comment by tleperou** [comment]
:raised_hands: 

Here's a [Stackblitz of it](https://stackblitz.com/edit/react-ts-gg9mxv?file=App.tsx,app.css,react-tinder-card.js%3AL147).

Within my actual implementation -- several pictures throughout a slider per card, I observed an improvement. Specially when playing with dragging the cards in the viewport -- and its edge.

It's compatible but, still a few chances of not being supported remain: see [caniuse](https://caniuse.com/requestanimationframe).

**Comment by 3DJakob** [comment]
I have now completely remade the web animation engine to use spring so this issue should be resolved in 1.6.0. See https://github.com/3DJakob/react-tinder-card/pull/115 for more information.
